### DISCUSSION TITLE
base url/path

### DISCUSSION THREAD

**Discussion by cpt-n3mo** [sveltejs/kit#9568] [Q&A]
Good day..

I was wondering if it were possible to prefix the routes 
for instance 

```
example.com/en/route
example.com/de/route
example.com/nl/route

```
this so that fetching from api in the right lang, and so that SEO is to googles specs.

kind regards.

**Comment by fedorovvvv** [comment]
Hi, you can wrap your routes in an [optional parameter](https://kit.svelte.dev/docs/advanced-routing#optional-parameters)
